maphandler.lua

```diff
@@ -754,7 +754,8 @@ function MapHandler:Init()
 	if DebugDrawEnabled then
 		self.map:EraseAll(4, 5)
 	end
-
+	ConUnitPerTypeLimit = math.max(map:SpotCount() / 6, 4)--add here cause map:spotcount not correctly load or so
+	ConUnitAdvPerTypeLimit = math.max(map:SpotCount() / 8, 2)
```

Comment:
i don't think setting globals in a module like this will work for non-lua Shard. better to set `ai.conUnitPerTimeLimit`. i think ultimately i should move stuff like this, from `CheckForMapControl` and `CheckMySide` to a new module